### PR TITLE
refactor landing page

### PR DESCRIPTION
**Pull request by mikeshng**
- modernize the look and feel (with least amount of work as possible)
- update hugo theme
- updated logo
- emphasis more content on core foundation apis such as registration, manifestwork and placement
- removed contents that are no longer valid in community (ui, install, clc etc.)
- less of a "doc" upon entry.

The entry page now looks like the following:

image

mobile version:

image

clicking the quick start will bring you to

image

Signed-off-by: Mike Ng <ming@redhat.com>

### CODE REVIEW

**Comment by qiujian16** [comment]
should we review it in one of the community meeting?

File: content/_index.md [hunk 1]
```diff
@@ -1,33 +1,55 @@
 ---
 title: What is Open Cluster Management?
+description: Open Cluster Management is the best way to simplify management of multiple Kubernetes clusters.
+geekdocNav: false
+geekdocAlign: center
+geekdocAnchor: false
 ---
 
-The Open Cluster Management project consists of several multicluster components, which are used to access and manage your clusters. It was started to bring together the various aspects of managing Kubernetes clusters into an integrated solution comprised of loosely coupled building blocks. Here are some examples of the Multicluster APIs that are part of the project today:
+The Open Cluster Management is a community-driven project focused on multicluster and multicloud scenarios for Kubernetes apps. Open APIs are evolving within this project for cluster registration, work distribution, dynamic placement of policies and workloads, and much more.
```
Comment (swopebe) [inline content/_index.md:9]: Open Cluster Management is a community-driven project focused on multicluster and multicloud scenarios for Kubernetes apps. Open APIs are evolving within this project for cluster registration, work distribution, dynamic placement of policies and workloads, and much more.

small change here, just eliminate the `The`

File: content/_index.md [hunk 2]
```diff
@@ -1,33 +1,55 @@
...
+## Feature overview
 
-![open-cluster-management](/ocm-small.png)
+{{< columns >}}
 
-Install and create a [Cluster manager](https://operatorhub.io/operator/cluster-manager) on your _hub_ cluster.
+### Cluster registration
 
-Install and create a [Klusterlet agent](https://operatorhub.io/operator/klusterlet) on your _manage_ cluster.
+Registration of multiple clusters to a hub cluster to place them under management.
```
Comment (swopebe) [inline content/_index.md:19]: let's say "for management" -- under is directional and can be an accessibility hit.

File: content/_index.md [hunk 3]
```diff
@@ -1,33 +1,55 @@
...
 
-For more details see [Core Components](getting-started/core).
+<--->
 
-## Governance policy framework
+### Work distribution
 
-Enable the Policy framework add-on to gain visibility and drive remediation for various security and configuration aspects to help meet such enterprise standards. For more details see [Policy framework add-on](getting-started/integration/policy-framework).
+Work API that enables resources to be applied to managed clusters from a hub cluster.
```
Comment (swopebe) [inline content/_index.md:25]: SO here you need the THE. The work API....

File: themes/hugo-geekdoc/CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
+# Contributing
+
+## Security
+
+If you think you have found a **security issue**, please do not mention it in this repository.
```
Comment (swopebe) [inline themes/hugo-geekdoc/CONTRIBUTING.md:5]: Confused about this, I thought we ironed out a common doc for this one? If we are changing that, this needs just a bit of work.

**Review by swopebe — CHANGES_REQUESTED** [review]
see comments

File: themes/hugo-geekdoc/CONTRIBUTING.md (hunk 4 above)
Comment (mikeshng) [reply themes/hugo-geekdoc/CONTRIBUTING.md:5]: This is under `themes` and not something we will display.

File: content/_index.md (hunk 3 above)
Comment (mikeshng) [reply content/_index.md]: Thanks. Fixed in https://github.com/open-cluster-management/website/pull/78/commits/e1f34f0967b8fe5d9c6092b69e249d65c77a10d9

File: content/_index.md (hunk 2 above)
Comment (mikeshng) [reply content/_index.md]: Thanks. Fixed in https://github.com/open-cluster-management/website/pull/78/commits/e1f34f0967b8fe5d9c6092b69e249d65c77a10d9

File: content/_index.md (hunk 1 above)
Comment (mikeshng) [reply content/_index.md]: Thanks. Fixed in https://github.com/open-cluster-management/website/pull/78/commits/e1f34f0967b8fe5d9c6092b69e249d65c77a10d9

**Review by swopebe — APPROVED** [review]

**Pull request merged**